### PR TITLE
New version: Microsoft.WindowsTerminal version 1.18.2681.0

### LINKED ISSUE #108419: [Update Request]: Windows Terminal
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Microsoft.WindowsTerminal

### Package Version

1.17.11461.0

### Please describe the changes you would like to see

There is a new release 1.17.11461.0 available.
https://github.com/microsoft/terminal/releases

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

* Resolves #108419
* Resolves #120903
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120906)

### CODE REVIEW

File: manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Scope: user
+InstallModes:
+- silent
+Commands:
+- wt
+PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.UI.Xaml.2.8
+      MinimumVersion: 2.8.4
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml:19]: Weren't we meant to change this? I remember @yao-msft's PR had changes requested and got closed as stale
- https://github.com/microsoft/winget-pkgs/pull/111968

**Comment by mdanish-kh** [comment]
This is probably failing due to `MinimumVersion` not being enabled in the pipelines

**Comment by DHowett** [comment]
Hey there! This looks like a duplicate of https://github.com/microsoft/winget-pkgs/pull/120903

**Comment by DHowett** [comment]
I think yours is better though...!

File: manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml (hunk 1 above)
Comment (DHowett) [reply manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml:19]: this is the actual min version we depend on in Terminal; is there something we need to do differently?

Comment (DHowett) [reply manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml:19]: oh i see, this versus 8.xxxx.yyyy.zzzz

Comment (Trenly) [reply manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml:19]: I opened #120909 to replace the one Yao had that was stale; 
The `8.2306.22001.0` change is needed for winget to detect that it is installed.

The `2.8.4` will work both before and after #120909 is merged, but should be updated afterwards

**Comment by Trenly** [comment]
> This is probably failing due to `MinimumVersion` not being enabled in the pipelines

Actually, I think the pipelines are still on WinGet 1.4

**Comment by Trenly** [comment]
> Hey there! This looks like a duplicate of #120903

That it is - but intentionally so; Since 1.6 was released, we are waiting for the 1.6 client to hit the pipelines and for dependencies validation to be enabled. Since Terminal is such a popular package, I would hope that someone like @stephengillie would be able to manually validate the package, although I know that can't be done for every single package that has dependencies.

Rather than try to take that PR and figure out if it is correct or needs modification or what the deal is, when working with new features it is easier to make a new PR that references and closes the old one. This way I know that the content should pass a manual validation but it still keeps the history of the previous PR

**Review by denelon — APPROVED** [review]

**Comment by denelon** [comment]
@Trenly if you can share your screenshot of this working with the dependencies, I'll merge it.

**Comment by Trenly** [comment]
> @Trenly if you can share your screenshot of this working with the dependencies, I'll merge it.

image

@denelon

**Pull request merged**